airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/destination/DestinationAcceptanceTest.kt

```diff
@@ -90,7 +90,7 @@ abstract class DestinationAcceptanceTest {
     protected var localRoot: Path? = null
     open protected var _testDataComparator: TestDataComparator = getTestDataComparator()
 
-    open fun getTestDataComparator(): TestDataComparator {
+    open protected fun getTestDataComparator(): TestDataComparator {
```

Comment:
I hope we have some formatter rules in Kotlin land to fix these kind of mismatches .. `protected open` vs `open protected`

Reply:
I think there's a compiler warning against that. Right now I had to disable them but will eventually re-enable them as errors